### DISCUSSION TITLE
Clone object

### DISCUSSION THREAD

**Discussion by Curium1** [formkit/formkit#1312] [Ideas]
Hi,
I have not seen any way to clone an element from one list to another?
Is that possible?

**Comment by sashamilenkovic** [comment]
@Curium1 Just to be clear, are you referring to @formkit/drag-and-drop?

**Reply by Curium1** [reply]
That is correct :)

**Reply by sashamilenkovic** [reply]
@Curium1 Haha, just checking. Feel free to post in @formkit/drag-and-drop in the future. There is an open issue about it with a workaround to do this: https://github.com/formkit/drag-and-drop/issues/8. Let me know if you have any questions

**Reply by Curium1** [reply]
Nice! Thank you :)
